src/Integration/Binding/ProjectBindingOperation.cs

```diff
@@ -172,7 +172,10 @@ private void AddFileToProject(Project project, string fullFilePath)
             var projectSystem = this.serviceProvider.GetService<IProjectSystemHelper>();
             projectSystem.AssertLocalServiceIsNotNull();
 
-            if (!projectSystem.IsFileInProject(project, fullFilePath))
+            // Workaround for bug https://github.com/SonarSource/sonarlint-visualstudio/issues/881
```

Comment:
it would be nice to mention that this file does not need to be added to the project in the 2019 system type